Makefile

```diff
@@ -46,7 +46,7 @@ ifdef SHOULD_LINT
 	@echo "Installing test dependencies for vet..."
 	@go test -i $(PKGS)
 	@echo "Checking vet..."
-	@$(foreach dir,$(PKG_FILES),go tool vet $(VET_RULES) $(dir) 2>&1 | tee -a lint.log;)
+	@go vet ./... 2>&1 | tee -a lint.log;
```

Comment:
I think `./...` is preferable. The Go tooling handles vendor/ exactly how we
want it to, so we don't need to rely on `glide nv`, especially since it won't
exist once we switch this library to Go modules.

There's a follow-up task here to simplify the Makefile but I'm going to hold
off on that for now.